app/controllers/slug_migrations_controller.rb

```diff
@@ -46,4 +46,11 @@ def update
   def show
     @slug_migration = SlugMigration.find(params[:id])
   end
+
+  def delete_search_index
+    @slug_migration = SlugMigration.find(params[:slug_migration_id])
+    @slug_migration.delete_search_document!
+
+    redirect_to edit_slug_migration_path(@slug_migration), notice: "Search index has been deleted"
```

Comment:
As Richard has mentioned, we should say: "Document has been deleted from the search index" or something similar